tests/installation/user_settings.pm

```diff
@@ -18,33 +18,33 @@ use testapi;
 
 sub run {
     my ($self) = @_;
-    assert_screen "inst-usersetup";
+    assert_screen 'inst-usersetup';
     if (get_var 'ROOTONLY') {
         send_key 'alt-s';
         assert_screen 'inst-rootonly-selected';
         # done user setup
         send_key $cmd{next};
+        return;
+    }
+    
```

Comment:
@okurz what's the point of "please run make test" comment if you merge unclean code anyway?

Reply:
You mean you like nested else wrappers that go for 20 lines of code?

Reply:
@kravciak It's a reminder for the next PR and I trusted @rwx788 

@rwx788 shame on you! ;)

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4442